packages/blade/src/components/Typography/Text/__tests__/Text.native.test.tsx

```diff
@@ -46,6 +46,35 @@ describe('<Text />', () => {
     expect(toJSON()).toMatchSnapshot();
   });
 
+  it('should render Text with variant "body" and size "small"', () => {
+    const displayText = 'Displaying some text';
+    const { toJSON, getByText } = renderWithTheme(
+      <Text type="normal" variant="body" weight="bold" truncateAfterLines={3} size="small">
+        {displayText}
+      </Text>,
+    );
+    expect(getByText('Displaying some text')).toBeTruthy();
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should throw error when variant is "caption" and size "small" is passed', () => {
+    try {
+      const displayText = 'Displaying some text';
+      renderWithTheme(
+        // @ts-expect-error testing failure case when size='small' is passed with variant='caption'
+        <Text type="normal" variant="caption" truncateAfterLines={3} size="small">
+          {displayText}
+        </Text>,
+      );
+    } catch (error: unknown) {
+      if (error instanceof Error) {
```

Comment:
Never noticed this. But ideally we should not call expect() conditionally. 

Say introduced a custom Error reported. `BladePropError()` this check will not pass.

Reply:
if we don't add this check TS yells because it doesn't know what is error's type 😭

Reply:
Yeah just set any.